README.md

```diff
@@ -51,6 +51,8 @@ Integration (CI) suite running in [travis ci](travis-ci.org).  This currently co
 
    Status: [![Build Status](https://travis-ci.org/openrisc/or1k_marocchino.svg?branch=master)](https://travis-ci.org/openrisc/or1k_marocchino)
 
+[Librecores-CI](https://github.com/librecores/docker-images/tree/master/librecores-ci) docker image is used for testing and continous integration. Librecores CI image includes many common EDA tools such as Icarus Verilog, Verilator, Yosys and also target the FuseSoC use cases. In the travis flow `or1k-test` runs inside a Librecores-CI docker container.
```

Comment:
Maybe you should just look to.
https://github.com/librecores/docker-images

Can you split this line up a bit. Try to limit a line to about 80 characters.  It sould be consistent with other blocks in the readme.

Maybe above mention "... suite running a librecores-ci docker container in Travis ci...."

Reply:
Did the changes. Please Let me know if it is okay. Thank you